sklearn/ensemble/tests/test_forest.py

```diff
@@ -389,7 +389,7 @@ def check_oob_score(name, X, y, n_estimators=20):
         assert abs(test_score - est.oob_score_) < 0.1
```

Comment:
Indeed we are already doing this for the classification. I think it makes sense to do that for the regression.
We only require a comment to mention that in the first case, this is a diff between accuracies and in the second one a diff between R2.